### PR TITLE
[DOCFIX] Update the default client jar path to hadoop

### PR DESCRIPTION
**Pull request by yupeng9**

### CODE REVIEW

**Comment by yupeng9** [comment]
jenkins, test this please

File: docs/_config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ ALLUXIO_RELEASED_VERSION: 1.5.0
 ALLUXIO_MASTER_VERSION_SHORT: 1.5.0
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro defined above.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/core/client/runtime/target/alluxio-core-client-runtime-1.5.0-jar-with-dependencies.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/hadoop/alluxio-1.5.0-hadoop-client.jar
```
Comment (calvinjia) [inline docs/_config.yml]: Call this `ALLUXIO_CLIENT_JAR_PATH_HADOOP`?

File: docs/_config.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ ALLUXIO_RELEASED_VERSION: 1.5.0
 ALLUXIO_MASTER_VERSION_SHORT: 1.5.0
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro defined above.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/core/client/runtime/target/alluxio-core-client-runtime-1.5.0-jar-with-dependencies.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/default/alluxio-1.5.0-default-client.jar
```
Comment (gpang) [inline docs/_config.yml]: Are only hadoop components using this path? Then should this be explicitly hadoop related?

`ALLUXIO_CLIENT_JAR_PATH_HADOOP: /<PATH_TO_ALLUXIO>/client/hadoop/alluxio-1.5.0-hadoop-client.jar`

Comment (yupeng9) [inline docs/_config.yml]: `alluxio-1.5.0-default-client.jar` is a symlink to `alluxio-1.5.0-hadoop-client.jar`

**Pull request merged**